Just.xcodeproj/project.pbxproj

```diff
@@ -186,7 +186,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0800;
-				LastUpgradeCheck = 0900;
```

Comment:
thanks again! can we get rid of all changes in this file, please?

Reply:
Done. Thanks for the quick reply!